dotcom-rendering/docs/architecture/3rd-party-technical-review/003-gracenote.md

```diff
@@ -15,7 +15,7 @@ We are due to use a third party, Gracenote, to supply us the data for our Olympi
 
 ## Data privacy engineering review
 
--   Google Tag Manager and google analytics included in tag script.
+-   Google Tag Manager included in tag script.
```

Comment:
I am not sure if all references to Google Analytics should be removed in this file or this file should left untouched.

Reply:
I will re-add the removed text 2e9820bc88da3017d3f071742342ead21acb414e based on this https://github.com/guardian/dotcom-rendering/pull/11470#discussion_r1626257779